packages/core/core/src/AssetGraphBuilder.js

```diff
@@ -818,7 +818,7 @@ export default class AssetGraphBuilder extends EventEmitter {
   }
 
   async readFromCache(): Promise<?Array<Event>> {
-    if (this.options.disableCache) {
+    if (this.options.shouldDisableCache) {
```

Comment:
I thought we had talked about this in the call but maybe this should rather be `shouldEnableCache`.

Reply:
All changes have been taken from the linked PR by Devon, which are the changes we agreed on I think.

Reply:
Yeah, but at the same time I remember Will saying that double negations should be avoided 🤷‍♂️

Reply:
Yeah it was commented out so probably not final yet